src/system.c

```diff
@@ -407,6 +409,6 @@ int sys_notify_id_valid(int fd, uint64_t id)
 		return -EOPNOTSUPP;
 
 	if (ioctl(fd, SECCOMP_IOCTL_NOTIF_ID_VALID, &id) < 0)
-		return -errno;
+		return -ENOENT;
```

Comment:
I didn't expect to see an `-ENOENT` error in this patch, but I think it's the right change to make for this ioctl.

Reply:
Not sure what to say in response here other than "yup" :)